src/config-schema.coffee

```diff
@@ -112,7 +112,16 @@ module.exports =
         description: 'Allow items to be previewed without adding them to a pane permanently, such as when single clicking files in the tree view.'
         type: 'boolean'
         default: true
-
+      telemetryConsent:
+        description: 'Allow usage statistics and exception reports to be sent to the Atom team to help improve the product.'
+        title: 'Send Telemetry to the Atom Team'
+        type: 'string'
+        default: 'undecided'
+        enum: [
+          {value: 'limited', description: 'Allow limited anonymous usage stats, exception and crash reporting'}
```

Comment:
There is no higher level of telemetry collection in Atom. The value is
called limited to make it clear in the future that consent was only given
for limited telemetry.

On Mon, Sep 5, 2016, 6:41 AM Jiri Spac <notifications@github.com> wrote:

> In src/config-schema.coffee
> <https://github.com/atom/atom/pull/12281#discussion_r77523588>:
>
> > @@ -112,7 +112,16 @@ module.exports =
> >          description: 'Allow items to be previewed without adding them to a pane permanently, such as when single clicking files in the tree view.'
> >          type: 'boolean'
> >          default: true
> > -
> > +      telemetryConsent:
> > +        description: 'Allow usage statistics and exception reports to be sent to the Atom team to help improve the product.'
> > +        title: 'Send Telemetry to the Atom Team'
> > +        type: 'string'
> > +        default: 'undecided'
> > +        enum: [
> > +          {value: 'limited', description: 'Allow limited anonymous usage stats, exception and crash reporting'}
>
> @damieng <https://github.com/damieng> why is there no option like:
> unlimited ? I want to make Atom better and I mean business.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/atom/atom/pull/12281/files/369bd12faaf656b94d01affa8b3d63d4796e4107#r77523588>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAHQp65yMhVVspLxaKwhKbnUw8XF_RCxks5qnBwWgaJpZM4JaPnm>
> .
>